examples/gcp-cloudrun/README.md

```diff
@@ -25,9 +25,10 @@ These values are indicated with `***`.
    pulumi -C infra config set gcp:region us-west1 # any valid GCP region here
    ```
 
-3. Authenticate to GCR with Docker:
+3. Authenticate to GCR with application-default credentials and Docker:
 
    ```bash
+   gcloud auth application-default login # this is required for pulumi to reach GCP
```

Comment:
this is covered under prerequisites but we can mention it here as well ;)

the thing is this is only one of the ways to authenticate to gcp, so it might be required, and probably will for newcommers